resources/views/docs/_structures/kudosu_history.md

```diff
@@ -3,7 +3,7 @@
 Field      | Type       | Description
 -----------|------------|-----------------------------
 id         | number     | |
-action     | string     | Either `give`, `reset`, or `revoke`.
+action     | string     | Either `vote.give`, `vote.reset`, or `vote.revoke`.
```

Comment:
the non prefixed ones still exist

Reply:
have changed it to reference all 6 possibilities in that case, reads a bit wonky but not sure there's a better wording